index.html

```diff
@@ -0,0 +1,74 @@
+    <link rel="stylesheet" type="text/css" href="css/libs/bootstrap.min.css">
+    <link href='http://fonts.googleapis.com/css?family=Raleway|Indie+Flower' rel='stylesheet' type='text/css'> <!--Google Fonts-->
+    <link rel="stylesheet" type="text/css" href="css/libs/style.css"> <!--Icon Fonts-->
+</head>
+<body>
+    <div  class="container-fluid">
+        <div class="row">
+            <div class="col-md-1"></div>
+            <div class="col-md-10">
+
+                <header>
+                    <div class="row">
+                        <div class="col-md-6 cat-container">
+                            <img src="images/cat.png" class="img-responsive img-circle cat" alt="surprised cat">
+                        </div>
+                        <div class="col-md-6">
+                            <h1 class="text-right">MARVIN ABEL</h1>
+                            <h4 class="text-right">FRONT-END JōNIN</h4>
+                        </div>
+                    </div>
+                </header>
+
+                <div> <!--Main image-->
+                    <img src="images/ming.png" class="img-responsive img-rounded dog" alt="dog">
+                </div>
+
+                <section> <!--Images section-->
+                    <h2 class="featured-work">Featured Work</h2>
+                    <div class="row">
+
+                        <div class="col-md-4 pictures-featured-work">
+                            <figure>
+                                <img src="images/baby.jpg" class="img-thumbnail" alt="baby">
+                                <figcaption>BABY</figcaption>
+                            </figure>
+                            <span class="icon-facebook2"></span><a href="https://www.facebook.com/Marbel.MP">Facebook</a>
+                        </div>
+
+                        <div class="col-md-4 pictures-featured-work">
+                            <figure>
+                                <img src="images/girl.jpg" class="img-thumbnail" alt="girl">
+                                <figcaption>GIRL</figcaption>
+                            </figure>
+                            <span class="icon-google-plus2"></span><a href="https://plus.google.com/u/0/103779539213798147419/posts">Google +</a>
+                        </div>
+
+                        <div class="col-md-4 pictures-featured-work">
+                            <figure>
+                                <img src="images/selena.jpg" class="img-thumbnail" alt="selena">
+                                <figcaption>TEEN</figcaption>
+                            </figure>
+
+                            <span class="icon-github"></span><a href="https://github.com/AbelMon">Git</a>
+                        </div>
+
+                    </div> <!--End of .row-->
+                </section>
+
+            </div> <!--End of col-md-10-->
+
+            <div class="col-md-1"></div>
+
+
+        </div> <!--End of .row-->
+    </div> <!--End of .container-fluid-->
+</body>
+</html>
```

Comment:
#### Awesome :white_check_mark: 

Your code looks very clean and easy to read, thanks! Using indentations really makes a huge difference in the code readability.